Fix out-of-bounds index which caused crash on arm64 systems.
Reported to the developer by email.
--- src/sort.c	(revision 650)
+++ src/sort.c	(working copy)
@@ -514,8 +514,8 @@
 	lname[0]='\0';
 	*reordered=0;
 	
-	utf16le_lname[MAX_PATH_LEN*4]='\0';
-	utf16le_lname[MAX_PATH_LEN*4+1]='\0';
+	utf16le_lname[MAX_PATH_LEN*2]='\0';
+	utf16le_lname[MAX_PATH_LEN*2+1]='\0';
 	
 	while (chain != NULL) {
 		device_seekset(fs->device, getClusterOffset(fs, chain->cluster));
